src/js/descript.js

```diff
@@ -135,5 +167,41 @@
         return containerName ? this._containers[containerName] : this._containers;
     };
 
+    /**
+     * Replaces string patterns in inline scripts. Patterns can be
+     * @param containerName
+     * @param scriptAttribute
+     * @param pattern
+     * @param replacement
+     */
+    Descript.prototype.replace = function(containerName, scriptAttribute, pattern, replacement) {
+        var containerScripts = this.get(containerName).toArray();
+        var script = this._find(containerScripts, scriptAttribute);
```

Comment:
It did originally, but I wanted to change it because `injectScript` gets the container too, so I felt like it was redundant to do it twice. Seeing as `get` is basically an accessor for getting the object property it's probably not too expensive of a call. I'll change it.